app/code/Magento/Catalog/Model/Category/FileInfo.php

```diff
@@ -169,14 +169,20 @@ public function isBeginsWithMediaDirectoryPath($fileName)
     /**
      * Get media directory subpath relative to base directory path
      *
+     * @param string $filePath
      * @return string
      */
-    private function getMediaDirectoryPathRelativeToBaseDirectoryPath()
+    private function getMediaDirectoryPathRelativeToBaseDirectoryPath(string $filePath = '')
     {
         $baseDirectoryPath = $this->getBaseDirectory()->getAbsolutePath();
         $mediaDirectoryPath = $this->getMediaDirectory()->getAbsolutePath();
 
         $mediaDirectoryRelativeSubpath = substr($mediaDirectoryPath, strlen($baseDirectoryPath));
+        $pubDirectory = 'pub/';
```

Comment:
It would be better to use constant `\Magento\Framework\App\Filesystem\DirectoryList::PUB` in this line.

Reply:
@dmytro-ch done